coverage-experiment/src/main/java/nl/tudelft/serg/evosql/experiment/Runner.java

```diff
@@ -61,15 +112,82 @@ public static void main(String[] args) {
      * @param connectionDataProd data of db connection for production database
      * @param connectionDataTest data of db connection for test database
      */
-    public static QueryExperimentResult runForQuery(String query,
-                                             ConnectionData connectionDataProd,
-                                             ConnectionData connectionDataTest,
-                                             String queryClassName) {
+    public static void runForQuery(String query,
+                                   ConnectionData connectionDataProd,
+                                   ConnectionData connectionDataTest,
+                                   int queryIndex) {
 
-        // TODO: Implement method
+        String packageName = "query" + queryIndex;
+        Path projectPath = Paths.get(EXPERIMENT_PATH.toString(), String.valueOf(queryIndex));
 
-        return null;
-    }
+        Path testClassPath = null;
+        try {
+            testClassPath = GradleUtil
+                    .setupGradleTestProject(projectPath, packageName);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        // Execute brew and output to project folder for original
+        BrewExecutor brewExecutor = new BrewExecutor(connectionDataProd, connectionDataTest, query, packageName);
+        brewExecutor.executeBrew(brewExecutor.getQueryResult(),
+                testClassPath, ORIGINAL_NAME);
+
+        // Create mutants
+        QueryMutator queryMutator = new QueryMutator(query, connectionDataProd);
+        List<String> queryMutants = new ArrayList<>();
+        try {
+            queryMutants = queryMutator.createMutants();
+        } catch (MutationException e) {
+            e.printStackTrace();
```

Comment:
Maybe we should cascade this exception so that it is caught in the main method and gets output through the logger.

Reply:
👍